src/plugins/status_page/public/status_page.js

```diff
@@ -36,6 +36,14 @@ const chrome = require('ui/chrome')
         ui.serverState = overall.state;
         ui.serverStateMessage = overall.title;
       }
+
+      // Unless the status page was explicitly requested, there is no
+      // reason to stay on the status page if the status is green; reload
+      // the window so the user is shown the page they were requesting.
+      const statusPageUrl = chrome.addBasePath('/status');
+      if ((overall.state === 'green') && ($window.location.pathname !== statusPageUrl)) {
+        return $window.location.reload();
+      }
```

Comment:
Thanks @spalger. Are you okay with me filing a separate issue to fix this, rather than adding to the scope of this PR? I will, of course, remove this reloading code block from this PR.